lab4/src/src/actions/other_info.js

```diff
@@ -0,0 +1,66 @@
+import GitHubAPI from '../services/githubAPI';
+import store from '../index';
+import React from 'react';
+
+export const getRepos = () => (dispatch) => {
+    if (store.getState().userInfo.reposUrl){
+        GitHubAPI.getOtherInfo(store.getState().userInfo.reposUrl).then(
+            (res) => {
+                const result = [];
+                res.forEach((element) => {
+                    result.push(element.name)
+                });
+                const repos = result.map((element, step) =>{
```

Comment:
it's bad practice to create react elements inside actions. Please leave rendering logic inside react components only.